models/articles_model.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection");
+
+exports.selectArticlesById = async (article_id) => {
+  if (isNaN(Number(article_id))) {
+    return Promise.reject({ status: 400, msg: "Bad Request" });
+  }
```

Comment:
If article_id is not a number then you can let the database reject it instead of doing it manually here. 

If you tried to query the database with the wrong data type, it will return an error. See what happens if you remove the above code and console log the error in the catch block. You will have an error with a code attached that you can use in your error handler. If you set this up correctly, you will be able to use it again for other endpoints too!